### ISSUE TITLE
Uncaught UpgradeRequiredError:  upgradeRequired_msg

### ISSUE THREAD

**Issue by ganthern** [tutao/tutanota#5694]
* send a calendar invitation to a free user
* download the .ics file from the invitation
* import that file into the personal calendar
* open the details popup and click any participation decision
* see uncaught error

## Test Notes
* [ ] bug does not reproduce anymore (instead, the status is reset and the upgrade dialog shows)
* [ ] if the user does not upgrade, nothing happens and the action can be re-attempted with the same result
* [ ] if the user upgrades, nothing happens and the action can be re-attempted and succeeds
* [ ] accepting invites from the mailbox still works as expected (for free and premium users, respectively)
* [ ] test this also with getting the invite by being premium and accepting it, then downgrading (the calendar should still show the event as an invitation with the buttons).

**Comment by paw-hub** [comment]
A bit of a weird thing I found when testing another issue after downgrading but then another user responds to the invite.

On one email:
![image](https://github.com/tutao/tutanota/assets/104824185/92934f0c-be04-42d5-94ee-bbf30d5769ee)

On another email:
![image](https://github.com/tutao/tutanota/assets/104824185/2a842317-d314-4995-b763-fe94edcc1dc3)

When restarting Tutanota:
![image](https://github.com/tutao/tutanota/assets/104824185/c06c7934-57a2-4aa0-b90b-4ec448d17818)

**Comment by vitoreiji** [comment]
I don't get what the weird thing is, can you please explain?

**Comment by paw-hub** [comment]
> I don't get what the weird thing is, can you please explain?

Basically, on a user that has replied to an event and then immediately downgraded to Free, I found that, if it receives an email that updates an event (e.g. someone was added as a participant), the update email will say that you have replied to the event, while the original email will give you the option to reply.

@ganthern and I talked about it in person and decided it's not anything particularly noteworthy.
